src/Editable/WebData.elm

```diff
@@ -0,0 +1,124 @@
+module Editable.WebData
+    exposing
+        ( EditableWebData(..)
+        , map
+        , notAskedReadOnly
+        , state
+        , toEditable
+        , toWebData
+        )
+
+{-| An EditableWebData represents an Editable value, along with WebData.
+
+`EditableWebData` is a wrapper type around [Editable](http://package.elm-lang.org/packages/stoeffel/editable/latest)
+and [WebData](http://package.elm-lang.org/packages/krisajenkins/remotedata/latest)
+
+It is used in order to keep track of the state of the Editable upon saving. That is,
+as we change teh `Editable` toEditable, and send it to the backend, we can keep track of their status
+(e.g. `RemoteData.Success` or `RemoteData.Failure`).
+
+@docs EditableWebData, notAskedReadOnly, map, toEditable, state, toWebData
+
+-}
+
+import Editable exposing (Editable(..))
+import RemoteData exposing (RemoteData(..), WebData)
+
+
+{-| A wrapper for `Editable`, that allows provides the means to track saving
+back to the backend via `WebData`.
+
+    import Editable
+
+    view : EditableWebData String -> Html msg
+    view editableWebData =
+        let
+            toEditable =
+                Editable.WebData.toEditable |> Editable.value
+
+            toWebData =
+                Editable.WebData.toWebData
+        in
+        text <| "Editable value is: " ++ toString toEditable ++ " with a WebDataValue of " ++ toString toWebData
+
+-}
+type EditableWebData a
+    = EditableWebData (Editable a) (WebData ())
+
+
+{-| Creates a new `EditableWebData`.
+-}
+notAskedReadOnly : a -> EditableWebData a
```

Comment:
hmmm... example would be great! I like `notAskedReadOnly`, because it's clear from the name what the initial state is.